### DISCUSSION TITLE
Navbar.Link component is hard loading the page while navigating

### DISCUSSION THREAD

**Discussion by KhatriFaiz** [heroui-inc/heroui#1019] [Help]
I was trying the Next UI first time with NextJS 13. While using "Navbar.Link" component inside Navbar component, It is not working. It is hard Loading the webpage.

When I used Link component of next js, It did work fine. it navigated without reloading the whole page.

I don't where I'm doing it wrong. Here's my code:

File Structure (according to the new Next JS 13 file structure):
```
App
| layout.jsx
| page.jsx
| head.jsx
| dashboard
    | page.jsx
    | Layout.jsx 
```
Code in app/Layout.jsx
```
"use client";
import { createTheme, Navbar, NextUIProvider, Text } from "@nextui-org/react";

const theme = createTheme({
  type: "dark",
  theme: {
    colors: {
      primary: "#4ADE7B",
      secondary: "#F9CB80",
      error: "#FCC5D8",
    },
    fonts: {
      sans: "-apple-system, BlinkMacSystemFont, 'Segoe UI'",
    },
  },
});

export default function RootLayout({ children }) {
  return (
    <html>
      <head />
      <body>
        <NextUIProvider theme={theme}>
          <Navbar>
            <Navbar.Brand>
              <Text h3>The Logo</Text>
            </Navbar.Brand>
            <Navbar.Content>
              <Navbar.Link href="/">Home</Navbar.Link>
              <Navbar.Link href="/dashboard">Dashboard</Navbar.Link>
            </Navbar.Content>
          </Navbar>
          {children}
        </NextUIProvider>
      </body>
    </html>
  );
}
```

Code in dashboard/layout.jsx:
```
"use client";

export default function DashboardLayout({ children }) {
  return <main>{children}</main>;
}
```

**Comment by Kismet333** [comment]
I've had the same problem for some time now. I don't think you are doing anything wrong, I think just the implementation of "next/link" has not been incorporated with the use of NextUI's <Navbar> component. Correct me if I am wrong NextUI team, but this is my current guess.

 I did attempt to solve the problem by adding a "next/link" component within the <Navbar.Link> component as can be seen in the example below:

```
import Link from "next/link";


<Navbar.Link href="/">
    <Link href="/">
        Home
    </Link>
</Navbar.Link>

```
This implementation throws the following error when first loading the page:
Error: Hydration failed because the initial UI does not match what was rendered on the server.

However, aside from that error and some slight styling weirdness, it appears to function correctly after that, using the single page approach and avoiding a hard reload. I have heard that downgrading React to v17.0.2 might prevent this error from being thrown as can be see here:
https://github.com/vercel/next.js/discussions/35773#discussioncomment-2481247


This might be a temporary solution and also I am just reporting my findings.

**Reply by KhatriFaiz** [reply]
@Kismet333 Thank you for your insights.

Although this method may work as intended, using an anchor tag inside the anchor tag is strictly prohibited by W3.org. (Follow this [link](https://www.w3.org/TR/html401/struct/links.html#h-12.2.2)) So I suggest you do not use this method. I personally prefer to use the Link component from "next/link" alone without using Navbar.Link Component. I style the Link components as needed.

**Comment by yuziwe** [comment]
I encountered a similar issue and resolved it using the following approach:
~~~
import Link from 'nextjs/link';

<Navbar.Link as={Link} href="/">Home</Navbar.Link>
<Navbar.Link as={Link} href="/">About</Navbar.Link>
~~~
You can find a same solution in [react-bootstrap/issues/3944](https://github.com/react-bootstrap/react-bootstrap/issues/3944#issuecomment-551249099)

**Comment by jrgarciadev** [comment]
Hey guys sorry for the delay please upgrade to [V2](https://nextui.org/docs/guide/upgrade-to-v2).
